src/UserService.Mappers/Db/DbEntityImageMapper.cs

```diff
@@ -0,0 +1,34 @@
+﻿using LT.DigitalOffice.UserService.Mappers.Db.Interfaces;
+using LT.DigitalOffice.UserService.Models.Db;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace LT.DigitalOffice.UserService.Mappers.Db
+{
+  public class DbEntityImageMapper : IDbEntityImageMapper
+  {
+    public List<DbEntityImage> Map(List<Guid> imagesIds, Guid entityId)
+    {
+      List<DbEntityImage> result = new();
+
+      if (imagesIds == null || !imagesIds.Any())
+      {
+        result = null;
```

Comment:
return null

Reply:
there would be too many returns in method then